binzx/otomi

```diff
@@ -69,7 +74,25 @@ else
 fi
 otomi_branch_or_tag=${current_branch:-$otomi_version}
 otomi_version_used=${OTOMI_TAG:-$otomi_version}
-readonly otomi_tools_image="otomi/core:${otomi_version_used}"
+otomi_image="cli"
+[ -n "$IN_DOCKER" ] && otomi_image="core"
+otomi_tools_image="otomi/$otomi_image:$otomi_version_used"
+# backwards compatibility with pre core/cli split era
+if [ $otomi_image = 'cli' ] && ! docker_tag_exists otomi/cli $otomi_version_used; then
+  # check if master version exists
+  if docker_tag_exists otomi/cli master; then
+    otomi_version_used='master'
```

Comment:
Since we moved to main branch, the tag should be `main`, right ?

Reply:
you may commit inline with batch